tests/unit/charms/lte_core_interface/v0/dummy_provider_charm/src/charm.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+from ops.charm import CharmBase, RelationJoinedEvent
+from ops.main import main
+
+from lib.charms.lte_core_interface.v0.lte_core_interface import LTECoreProvides
```

Comment:
```suggestion
from charms.lte_core_interface.v0.lte_core_interface import LTECoreProvides
```
This might be misleading. It actually caused a problem in @ghislainbourgeois PR last sprint.

Reply:
Fixed.